### PR TITLE
Feat : Added archive and trash

### PR DESCRIPTION
**Pull request by srejitk**
This feature request adds the following:
1. Archive Page
2. Trash Page
3. Color Pallett
4. Label Input

### CODE REVIEW

File: src/components/EmptyState/EmptyState.module.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  align-items: center;
+  width: 344px;
+  height: 328px;
+}
+
+.icon_container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
```
Comment (nishasen) [inline src/components/EmptyState/EmptyState.module.css:13]: use util class here

File: src/components/HomePage/HomePage.jsx [hunk 2]
```diff
@@ -1,42 +1,90 @@
...
+    setNote({ ...note, [name]: value });
+  };
+  console.log(note);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    note.isEdited
+      ? updateNote(note, noteDispatch)
+      : addNote(note, noteDispatch);
+    console.log(note);
```
Comment (nishasen) [inline src/components/HomePage/HomePage.jsx:42]: remove console.log

File: src/contexts/Note/NoteReducer.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    trashedNotes: [],
+    pinnedNotes: [],
+  };
+  switch (type) {
+    case "GET_NOTE":
+      return { ...state, notes: payload };
+    case "GET_ARCHIVED_NOTES":
+      return { ...state, archivedNotes: payload };
+    case "ADD_NOTE":
+      return { ...state, notes: payload };
```
Comment (aishwarya-bm) [inline src/contexts/Note/NoteReducer.jsx:14]: Single case be made for all three as all are doing same action

**Review by nishasen — APPROVED** [review]
Looks good and clean. well organized code

**Review by aishwarya-bm — APPROVED** [review]
Looks all good with reusable components and clean readable code and error handling! Left a small suggestion.

**Review by krishanu2911 — APPROVED** [review]
No changes are required Well written code
Nicely Defined functions
Good to go👍🏻

File: src/contexts/Note/NoteReducer.jsx (hunk 3 above)
Comment (srejitk) [reply src/contexts/Note/NoteReducer.jsx:14]: Yes, thank you. Have corrected it.

File: src/components/EmptyState/EmptyState.module.css (hunk 1 above)
Comment (srejitk) [reply src/components/EmptyState/EmptyState.module.css:13]: Thank you for pointing this out, have corrected it in the latest commit.

File: src/components/HomePage/HomePage.jsx (hunk 2 above)
Comment (srejitk) [reply src/components/HomePage/HomePage.jsx:42]: Done! Thanks for pointing this out

**Pull request merged**